Cargo.toml

```
@@ -31,6 +31,8 @@ hex = "0.4.3"
 kbs-types = "0.5.3"
 log = "0.4.17"
 prost = "0.11.0"
+#regorus = "0.1.2"
+regorus = { git = "https://github.com/microsoft/regorus" }
```

We could configure regorus to support only those builtins that will be used by the policies. The way it is used currently, we might not even need the `arc` feature since neither the Engine instance nor the `Value` instances are shared between threads.

```suggestion
regorus = { git = "https://github.com/microsoft/regorus",  default-features=false, features=["arc"] }
```

---

This seems to work. Thanks for the advice.